opentelemetry-sdk/src/metrics/internal/sum.rs

```diff
@@ -19,89 +20,122 @@ use super::{aggregate::is_under_cardinality_limit, AtomicTracker, Number};
 pub(crate) static STREAM_OVERFLOW_ATTRIBUTES: Lazy<Vec<KeyValue>> =
     Lazy::new(|| vec![KeyValue::new("otel.metric.overflow", "true")]);
 
+/// Abstracts the update operation for a measurement.
+trait Operation {
+    fn update_tracker<T: 'static>(tracker: &dyn AtomicTracker<T>, value: T);
```

Comment:
Can we avoid the dyn dispatch here? This is metrics hot path, and we need to try and make it as fast as possible.

Reply:
Done